paasta_tools/frameworks/adhoc_scheduler.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, *args, **kwargs):
         if kwargs.get('service_config_overrides') is None:
             kwargs['service_config_overrides'] = {}
         kwargs['service_config_overrides']['instances'] = 1
+        if kwargs.get('staging_timeout') is None:
+            kwargs['staging_timeout'] = 1
```

Comment:
seems too low for a default? also there is another default in this PR - 60s, why are these values different?